### DISCUSSION TITLE
Is this library still being supported?

### DISCUSSION THREAD

**Discussion by SipanOhanyan** [mandrewcito/signalrcore#115] [General]
Is this library still being supported?
The last release is on Apr 6, 2022.

**Comment by kresimirfijacko** [comment]
I am also interested in the same question.
If you found some alternative, feel free to share.

**Reply by SipanOhanyan** [reply]
We have forked this project and fixed several bugs.
You can use our version, link below:
[signalrcore](https://pypi.org/project/signalrcore1)

**Reply by kresimirfijacko** [reply]
thank you
i see info on pypi that points to original repository
do you have your own repo on github ?

**Reply by SipanOhanyan** [reply]
https://github.com/lynxal/signalrcore

**Comment by SipanOhanyan** [comment]
We have forked this project and fixed several bugs.
You can use our version, link below:
[signalrcore](https://pypi.org/project/signalrcore1)

**Comment by mandrewcito** [comment]
I resumed development this year; last year it was paused for personal reasons.
